src/ExtensionSet.php

```diff
@@ -424,9 +433,16 @@ private function initExtensions(): void
         $this->unaryOperators = [];
         $this->binaryOperators = [];
 
+        $classes = [];
         foreach ($this->extensions as $extension) {
-            $this->initExtension($extension);
+            if ($extension instanceof ExtensionInterface) {
+                $this->initExtension($extension);
+            } else {
+                $classes[] = $extension;
```

Comment:
making `$this->extensions` contain a mix of extensions and of runtimes (but not all of them) looks weird to me

Reply:
if this is a way to solve the case of `getLastModified`, I'd rather add a way to make ExtensionSet aware of a **separate** list of runtimes needing to be checked (which would allow solving the case where we don't invalidate the cache today if you change the argument names in a runtime while it actually impacts the template compilation if you use named parameters in Twig, by making TwigBundle inject the name of _all_ runtimes in the list).